### PR TITLE
install cerberus on shift-stack deployed by jetpack

### PR DESCRIPTION
**Pull request by masco**

### CODE REVIEW

File: OCP-4.X/post-install-on-osp.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+#
+# intall cerberus and datasever on shiftstack
```
Comment (chaitanyaenr) [inline OCP-4.X/post-install-on-osp.yml:3]: We might want to document the role of this playbook and how it runs as a post install step after jetski. Thoughts?

File: OCP-4.X/post-install-on-osp.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#
+
+- name: post install tasks on OSP
+  hosts: orchestration
+  gather_facts: true
+  remote_user: "{{ orchestration_user }}"
+  vars_files:
+    - vars/post-install-on-osp.yml
+  roles:
```
Comment (chaitanyaenr) [inline OCP-4.X/post-install-on-osp.yml:12]: We might also want to add the steps to setup dittybopper to run mutable grafana with our perf/scale dashboards loaded to view the prometheus metrics during the test runs if it's not part of jetpack post-install.

File: OCP-4.X/post-install-on-osp.yml (hunk 1 above)
Comment (masco) [reply OCP-4.X/post-install-on-osp.yml:3]: good one. I have added the same.

File: OCP-4.X/post-install-on-osp.yml (hunk 2 above)
Comment (masco) [reply OCP-4.X/post-install-on-osp.yml:12]: I have added this in jetpack itself :P

Comment (chaitanyaenr) [reply OCP-4.X/post-install-on-osp.yml:12]: Great.

File: docs/ocp4_osp_post_install.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
+# OpenShift 4 on OpenStack Post Install Documentation
+
+The OpenShift 4 on OpenStack post install playbook is `OCP-4.X/post-install-on-osp.yml` and will install and setup some cluster management tools like cerberus on a cluster installed by [jetpack](). The Jetpack will deploy given OpenStack version and deployes given OpenShift on top of it. The Jetpack also creates infra and workload nodes for day 2 operations.
```
Comment (chaitanyaenr) [inline docs/ocp4_osp_post_install.md:3]: The jetpack url is missing.

Comment (masco) [reply docs/ocp4_osp_post_install.md]: my bad. good catch

**Pull request closed without merging**